src/analyticsTracking/event_schemas/common_definitions.schema.json 

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
get's generated manually and consumes by humans for documentation. I felt it was better to have in-context version-controlled documentation of events in the codebase rather than externalize them out-of-context in the Amplitude app.

Reply:
Using jsonschema for documentation seems very heavy handed for writing and not too valuable for reading. Is there a better alternative for docs?
I'm also not sure how accurate it's going to be as the events are going to change format depending on the scope, no?

Reply:
yeah maybe a bit heavy handed and not great for readability. The other approach suggested by Amplitude is to keep a google sheet with the taxonomy on it but I worry that approach would rapidly fall out of sync and be another piece of information to have to track down.